### ISSUE TITLE
Load DataSet

### ISSUE THREAD

**Issue by MiguelSteph** [MiguelSteph/Pratical-Guide-To-SVM#1]
- Load Dataset into Panda DataFrame
- Shuffle the training set and the test set also
